server/contentRouter.js

```diff
@@ -121,31 +121,35 @@ const createDeleteManyBase = root =>
     res.send({ data: {} });
   };
 
-module.exports = (methods = {}, root = __dirname) => {
-  const {
-    getMany = createGetManyBase(root),
-    getOne = createGetOneBase(root),
-    patchOne = createPatchOneBase(root),
-    patchMany = createPatchManyBase(root),
-    putOne = createPutOneBase(root),
-    deleteOne = createDeleteOneBase(root),
-    deleteMany = createDeleteManyBase(root),
-  } = methods;
-  const router = express.Router();
+module.exports = {
+  createGetManyBase,
+  createGetOneBase,
+  router: (methods = {}, root = __dirname) => {
+    const {
+      getMany = createGetManyBase(root),
+      getOne = createGetOneBase(root),
+      updateOne = createUpdateOneBase(root),
+      updateMany = createUpdateManyBase(root),
+      createOne = createCreateOneBase(root),
+      deleteOne = createDeleteOneBase(root),
+      deleteMany = createDeleteManyBase(root),
+    } = methods;
+    const router = express.Router();
 
-  router.get('/:contentType', getMany);
+    router.get('/:contentType', getMany);
 
-  router.get('/:contentType/:itemId', getOne);
+    router.get('/:contentType/:itemId', getOne);
 
-  router.patch('/:contentType/:itemId', uploadMiddleware, patchOne);
```

Comment:
@Qunapi, looks like `uploadMiddleware` got accidentally removed during conflict resolve. Could you please do a quick scan of you own PRs after creation? thanks